data/blog/tailgraft.mdx

```diff
@@ -0,0 +1,179 @@
+---
+title: Grafting Tailscale onto the Pi
+date: 2023-07-14
+authors: ['xe']
+summary: Skip all the overhead and get to using your Raspberry Pi ASAP with Tailscale.
+tags:
+  - Raspberry Pi
+  - Ubuntu
+  - cloud-init
+images: ['/images/tailgraft/social.jpg']
+---
+
+Imagine this scenario: you're setting up that Raspberry Pi you were
+finally able to get your hands on after years of them being literal
+unobtanium, you look at the device, and then at your cables in pure
+despair. The Raspberry Pi 4 has two HDMI ports, but they're not normal
+HDMI. They're micro-HDMI and you don't have adaptors. Oh, you can
+figure out the IP address in your DHCP table, and then you could go
+through and connect over SSH, get told you need to change your
+password, get kicked out, and then reconnect. Then you can get around
+to installing Tailscale or whatever.
+
+This does work, but it is insufficiently magic for our needs. What if
+you could just turn your Pi on and have it magically appear on your
+tailnet? Keep reading, because that's exactly what we're gonna cover
+today: grafting Tailscale into your Raspberry Pi with the magic of the
+cloud.
+
```

Comment:
Maybe here something like:

The automatic approach we're using today is set up for Mac and Linux, but Windows users, you can find [information about how to set this up manually] at the bottom of this post.